src/platform/mhv/downtime/components/MHVDown.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatDatetime, formatElapsedHours } from '../utils/date';
+
+function MHVDown({
+  appTitle = 'some of our health tools',
+  endTime,
+  startTime,
+}) {
+  const startString = startTime ? formatDatetime(startTime) : '';
+  const endString = endTime ? formatDatetime(endTime) : '';
+  const timeInterval = formatElapsedHours(startTime, endTime);
+
+  return (
+    <>
+      <va-alert class="vads-u-margin-bottom--4" status="error" uswds visible>
+        <h3 slot="headline">Maintenance on My HealtheVet</h3>
+        <p>
+          We&#x2019;re working on My HealtheVet. The maintenance will last{' '}
+          {timeInterval}. During this time, you may have trouble using{' '}
```

Comment:
Can timeInterval be undefined/empty if any of the times are missing?

Reply:
After moving `timeInterval` to MHVDowntime component, I gave it a default value of `'some time'` in case any of the date times are empty or missing or not actually date times